tests/ctl/core/test_audit.py

```diff
@@ -82,26 +79,6 @@ def test_rich_organization_passes_audit(
     assert audit_findings == 0
 
 
-@pytest.mark.unit
-def test_basic_data_use_fails_audit(
-    test_basic_data_use: List[DataUse],
-) -> None:
-    audit_findings = audit.audit_data_use_attributes(
-        test_basic_data_use[0], "test_system_name"
-    )
-    assert audit_findings > 0
```

Comment:
Why was this removed?

Reply:
i removed the underlying function `audit_data_use_attributes ` at https://github.com/ethyca/fides/pull/4502/files#diff-031470e9e61c6b3656444c36bd09b63b01a69ff60b0e2bb66a6a6dfcc2d833deL101-L116 because it was only auditing the deprecated attributes that we're now removing! let me know if i've misunderstood something.

thanks for flagging though because i think there are some more fixtures to get rid of in this file since they're no longer being used 👍

Reply:
ah great makes sense